### DISCUSSION TITLE
ability to set QInput text color

### DISCUSSION THREAD

**Discussion by andrisi** [quasarframework/quasar#14715] [Ideas / Proposals] [answered]
It's nice that we have a "dark" option with white text color, but would be nice to be able to set the text color of a QInput, just as you can set it's label's color, without fiddling with CSS, and trying to hoplessly target the html INPUT element. Thanks.

**Comment by pdanpdan** [comment, marked as answer]
you have `input-class` prop

**Reply by andrisi** [reply]
thanks!
